FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/demos/V2ToolTipActivity.kt

```diff
@@ -133,30 +141,47 @@ fun CreateToolTipActivityUI(context: Context) {
         Column {
             ListItem.Header(title = context.getString(R.string.menu_xOffset),
                 trailingAccessoryContent = {
-                    BasicTextField(value = xOffsetState.value,
-                        modifier = Modifier.testTag(TOOLTIP_ACTIVITY_X_OFFSET_TEXTFIELD_TAG),
-                        keyboardOptions = KeyboardOptions(keyboardType = KeyboardType.Number),
-                        onValueChange = { xOffsetState.value = it.trim() })
+                    Box(Modifier.widthIn(100.dp,150.dp)) {
+                        TextField(
+                            value = xOffsetState.value,
+                            onValueChange = { xOffsetState.value = it.trim() },
+                            modifier = Modifier.testTag(TOOLTIP_ACTIVITY_X_OFFSET_TEXTFIELD_TAG),
+                            keyboardOptions = KeyboardOptions(keyboardType = KeyboardType.Number)
+                        )
+                    }
                 }
             )
             ListItem.Header(title = context.getString(R.string.menu_yOffset),
                 trailingAccessoryContent = {
-                    BasicTextField(value = yOffsetState.value,
-                        modifier = Modifier.testTag(TOOLTIP_ACTIVITY_Y_OFFSET_TEXTFIELD_TAG),
-                        keyboardOptions = KeyboardOptions(keyboardType = KeyboardType.Number),
-                        onValueChange = { yOffsetState.value = it.trim() })
+                    Box(Modifier.widthIn(100.dp,150.dp)) {
+                        TextField(
+                            value = yOffsetState.value,
+                            onValueChange = { yOffsetState.value = it.trim() },
```

Comment:
have a question on this trim, when the user is typing then onValueChange triggers so if user adds a space after a word wouldn't that be remove 🤔  ? 

this should be done when we submit the forms.

Reply:
@rohanprasadofficial  This is an input field for numeric value. So not allowing using space to enter so that api can convert string to numeric value.

Reply:
got it 👍